icicle/appUtils/ntt/tests/verification.cu

```diff
@@ -41,7 +41,7 @@ int main(int argc, char** argv)
   cudaEvent_t icicle_start, icicle_stop, new_start, new_stop;
   float icicle_time, new_time;
 
-  int NTT_LOG_SIZE = (argc > 1) ? atoi(argv[1]) : 19;
+  int NTT_LOG_SIZE = (argc > 1) ? atoi(argv[1]) : 10;
```

Comment:
why do you change this in this pr?

Reply:
just wanted to debug faster and forgot to revert